### DISCUSSION TITLE
How to use automatic mode on background remover?

### DISCUSSION THREAD

**Discussion by Palmo23** [T8RIN/ImageToolbox#979] [Q&A] [answered]
Hello all,

I'm new to this software (btw loving it) but I was curious about the background remover and how to use the automatic function. It only lets me draw or use the lasso tool. I was thinking the "auto" setting was kind of like an automatic selection. Thank you guys in advance for the help.

**Comment by T8RIN** [comment, marked as answer]
Hi, it's not available in the Foss version unfortunately

**Reply by Palmo23** [reply]
Ok thank you for the clarification and quick response
